.github/workflows/test.yaml

```diff
@@ -63,4 +66,20 @@ jobs:
 
       - run: patchright install --no-shell chromium
 
-      - run: pytest tests/${{ matrix.test }}.py
+      - name: Install Playwright and Chromium
+        run: |
+          uv pip install playwright
+          uv run playwright install chromium
+          uv run patchright install chrome
```

Comment:
dont install `chrome`, that's the old headless which is no longer used by any code path
`chromium` is already installed directly above this section